modules/aioseop_sitemap.php

```diff
@@ -251,7 +251,7 @@ public function __construct() {
 			$this->prio     = array_merge( $arr_no, $prio );
 
 			$freq = array();
-			foreach ( array( 'always', 'hourly', 'daily', 'weekly', 'monthly', 'yearly', 'never' ) as $f ) {
+			foreach ( array( __( 'always', 'all-in-one-seo-pack' ), __( 'hourly', 'all-in-one-seo-pack' ), __( 'daily', 'all-in-one-seo-pack' ), __( 'weekly', 'all-in-one-seo-pack' ), __( 'monthly', 'all-in-one-seo-pack' ), __( 'yearly', 'all-in-one-seo-pack' ), __( 'never', 'all-in-one-seo-pack' ) ) as $f ) {
```

Comment:
Are these settings? If so, we don't want to translate option values.

Reply:
These can be translated but there's a small issue that has to be addressed first. I've opened https://github.com/semperfiwebdesign/all-in-one-seo-pack/issues/2401 for that. Once that issue is resolved, the plugin will be 100% translatable.